dask_sql/mappings.py

```diff
@@ -163,7 +163,8 @@ def sql_to_python_value(sql_type: str, literal_value: Any) -> Any:
         dt = datetime.fromtimestamp(
             int(literal_value.getTimeInMillis()) / 1000, timezone.utc
         )
-
+        if sql_type == "DATE":
```

Comment:
Would you mind adding a comment here linking to https://github.com/dask-contrib/dask-sql/issues/296?

```suggestion
        # address datetime format mismatch
        # https://github.com/dask-contrib/dask-sql/issues/296
        if sql_type == "DATE":
```